src/analyzer/protocol/ldap/ldap.spicy

```diff
@@ -410,6 +416,12 @@ type SaslCredentials = unit() {
   };
 };
 
+type SicilyMessage = unit(bat: BindAuthType) {
+    # Just ensure the signature matches. We could do more,
+    # but it'd be better to forward to an NTLM analyzer.
+    signature: b"NTLMSSP";
+};
```

Comment:
Suggest to drop the unit arg for now since it is unused. The current usage also always decodes the same string each time which carries unneeded costs. Maybe something like this (could even move `signature_decoded` to a global `const`)?

```ruby
type SicilyMessage = unit {
    # Just ensure the signature matches. We could do more,
    # but it'd be better to forward to an NTLM analyzer.
    : skip b"NTLMSSP";
    var signature_decoded: string = "NTLMSSP";
};
```